checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/session/request/handlers/PaymentsCvcSessionRequestHandler.kt

```diff
@@ -13,18 +13,32 @@ import com.worldpay.access.checkout.util.ValidationUtil.validateNotNull
 internal class PaymentsCvcSessionRequestHandler(
     private val sessionRequestHandlerConfig: SessionRequestHandlerConfig
 ): SessionRequestHandler {
-
+    /**
+     * Accepts a list of [SessionType]s and returns True if the list contains a [PAYMENTS_CVC_SESSION]
+     */
     override fun canHandle(sessionTypes: List<SessionType>): Boolean {
         return sessionTypes.contains(PAYMENTS_CVC_SESSION)
     }
 
+    /**
+     * Accepts in a [CardDetails] object and validates that the mandatory fields for this [SessionType] are present
```

Comment:
Don't need to say what it accepts here, just what it does, the `@param` annotation describes what it accepts already

Reply:
updated